synthetics/data_source_agents.go

```diff
@@ -44,12 +48,12 @@ func dataSourceAgents() *schema.Resource {
 				Type:     schema.TypeFloat,
 				Optional: true,
 			},
-			distanceKey: {
+			minDistanceKey: {
 				Type:     schema.TypeFloat,
```

Comment:
I just noticed that we have no documentation apart from example. Could you add Description field to newly added schema attributes?  It is then rendered into documentation. See: https://registry.terraform.io/providers/kentik/kentik-cloudexport/latest/docs/data-sources/item

BTW: please regenerate the documentation.
CI should probably check if the documentation is fresh. I shall create a task for that.

Reply:
Yup, every build should generate documentation if any of the inputs has changed.

Reply:
Addressed: 50c5ca970e8856030bc342b8ab98c109a9097939